tmt-app/src/Styling/CustomStyling.js

```diff
@@ -37,17 +37,17 @@ export const CustomTheme = createTheme({
       fontWeight: 700,
       fontSize: "64px",
       lineHeight: "96px",
-      "@media (max-width:1350px)": {
```

Comment:
Research into the theme palette object would be helpful in cleaning up some of these styles later, I think we've created some custom stuff and not leveraging MUI for what it is and what it's capable of